src/components/layout/Layout.module.css

```diff
@@ -11,7 +11,7 @@
   color: var(--on-background);
   height: 100%;
   width: 100%;
-  padding: 1em;
+  padding: 5em 3em 3em 3em;
```

Comment:
This was originally styled "mobile-first", so the 1em padding was for small screen sizes. Then the padding increased to 5em on 992px and higher. This current version would just be `5em 3em 3em 3em` on all screen sizes.